docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -51,3 +52,173 @@ described below, and relative to the joint TMREL.
+
+Because the relative risks from GBD are for all-cause mortality in the early and late neonatal period, we first define all-cause mortality rate (ACMR) as the sum of: 
+
+   - mortality from causes that are affected by lbwsg and modelled in the sim (green)
+   - mortality from causes that are affected by lbwsg but not modelled in the sim (blue)
+   - mortality from causes that are unaffected by lbwsg, modelled or not modelled (salmon)
+
+An example of these causes from the large-scale-food fortification concept model diagram is shown below: 
+
+.. image:: causes_equation.svg
+
+All-cause mortality is sum of all cause-specific mortalities: 
+
+   ACMR =  :math:`\sum\limits_{\text{green}}\text{CSMR} + \sum\limits_{\text{blue}}\text{CSMR} + \sum\limits_{\text{salmon}}\text{CSMR}`
+
+The mortality from unmodelled causes affected by lbwsg (blue) is thus: 
+
+   :math:`\sum\limits_{\text{blue}}\text{CSMR}` = ACMR - :math:`\sum\limits_{\text{salmon}}\text{CSMR} - \sum\limits_{\text{green}}\text{CSMR}`
+
+Because we model some of the causes affected by lbwsg, we can use their excess-morality rates (EMR) instead of the average CSMRs: The mortality from modelled causes affected by lbwsg (green):
+
+   - cause-specific mortality if the person who does NOT have the condition: 0
+   - cause-specific mortality if the person HAS the condition: EMR of the condition
+
+ 
+We are interested in applying the PAF and relative risk to only the causes that GBD considers to be affected by lbwsg (green and blue):
+
+   |  i = low birth weight short gestation category
+   |  mr_i = mortality hazard in early and late neonatal period for category i
+   |  rr_i = relative risk for all cause mortality in category i
+   |  state = either 1 with condition or 0 without condition 
+   |  PAF* = this is the PAF of the most detailed cause affected by lbwsg
+
+Hence, the mortality hazard for an individual in lbwsg category i is:  
+
+mr_i 
+
+   | = ACMR_i 
+   | = (sum of unaffected causes) + affected(sum of unmodelled + sum of modelled) x (1-PAF*) x :math:`rr_i`
+
+= :math:`\sum\limits_{\text{salmon}}\text{CSMR} + (\sum\limits_{\text{blue}}\text{CSMR} + \sum\limits_{\text{green}}\text{EMR_state})\cdot\text{(1-PAF*)}\cdot rr_i`
+
+= :math:`\sum\limits_{\text{salmon}}\text{CSMR} + (ACMR - \sum\limits_{\text{salmon}}\text{CSMR} - \sum\limits_{\text{green}}\text{CSMR} + \sum\limits_{\text{green}}\text{EMR_state})\cdot\text{(1-PAF*)}\cdot rr_i`
+
+
+.. important :: 
+
+   PAF in the above equation represents PAFs for most-detailed-cause (they are all roughly equal) affected by LBWSG (or as calculated in vivarium from LBWSG relative risks and exposure). This approach assumes that relative risks for LBWSG applies only to causes that GBD considers to be affected by LBWSG (green and blue causes). 
+
+.. todo ::
+   link notebook that ali made on obtaining most-detailed-cause PAF lbwsg PAFs and abie's notebook on continuous conversion of the categories 
```

Comment:
I think I have Abie's notebook somewhere - I'll look for it and make a PR in the vivarium_data_analysis repo if no one else has added it.

Reply:
ok thanks!

Reply:
@Ninicorn I just made a PR here for converting LBWSG categories into a continuous distribution: https://github.com/ihmeuw/vivarium_data_analysis/pull/42

Feel free to merge it into master so you can link to the notebook.